src/sailing_robot/src/sailing_robot/navigation.py

```diff
@@ -73,6 +74,12 @@ def wind_angle_to_heading(self, wind_angle):
         """
         return angleSum(self.absolute_wind_direction(), wind_angle)
     
+    def angle_average(self, angle_list):
+        """Compute the average angle of a list of angles (the result is % 360)
+        """
+        return math.degree(math.atan2(sum([ math.sin(math.radian(x)) for x in angle_list]),
+                                      sum([ math.cos(math.radian(x)) for x in angle_list]))) % 360
```

Comment:
This doesn't need to be a method (it never uses `self`). If you look at the end of this file, there's a group of standalone functions for angle calculations - stick it in there.